docs/slides/aws/terraform-community/index.md

```diff
@@ -445,12 +445,12 @@ class: title
 **Now that you have terraform installed and working with AWS, we can do a few dry runs before building real infrastructure. Follow along carefully, copying and pasting the commands on each slide into your shell as we go.**
 
 ---
-name: what-is-terraform-oss
+name: what-is-terraform-community
 class: img-left
 # What is Terraform?
 ![Terraform](images\Terraform_VerticalLogo_FullColor.png)
 
-Terraform is an open source provisioning tool.
+Terraform is a community provisioning tool.
```

Comment:
not sure of this makes sense grammatically

Reply:
Changed this in [479a90e](https://github.com/hashicorp/field-workshops-terraform/pull/419/commits/479a90ef131419bac7afbeba46e7023cb37b1012)